packages/core/src/abstract-dialect/query-generator-typescript.ts

```diff
@@ -607,6 +607,15 @@ export class AbstractQueryGeneratorTypeScript<Dialect extends AbstractDialect =
     };
   }
 
+  /**
+   * Returns the alias token 'AS' after `FROM` clause.
+   *
+   * @returns string
+   */
+  getAliasToken() {
```

Comment:
This should not be part of the public API. Can you move it to the relevant `QueryGeneratorInternal` classes?

Reply:
Moved to `QueryGeneratorInternal` classes.